pkg/object/httpserver/mux.go

```diff
@@ -404,11 +404,6 @@ func (m *mux) ServeHTTP(stdw http.ResponseWriter, stdr *http.Request) {
 				m.handleRequestWithCache(rules, ctx, ci)
 				return
 			}
-
```

Comment:
@qdongxu 
* In https://github.com/megaease/easegateway/pull/454 this PR, introducing `HTTPServer`'s header matching logic. Plz, conform to this remove behavior, I think the original implementation will cause the HTTP request be handled even in `header-not-match` scenario.

So I remove it for the continued processing logic below.